src/Scaffolding/VS.Web.CG.Mvc/Minimal Api/MinimalApiGenerator.cs

```diff
@@ -158,6 +158,11 @@ internal async Task AddEndpointsMethod(string membersBlockText, string endpoints
                         usings.Add(templateModel.DbContextNamespace);
                     }
 
+                    if (templateModel.OpenAPI)
+                    {
+                        usings.Add("Microsoft.AspNetCore.Http.HttpResults");
```

Comment:
Is there a reason `OpenApi` implies needing `HttpResults`?

Reply:
My guess is the fact that `OpenApi` implies the requirement to describe the API input/output shape. Previously this was done using the extension methods (`Produces` and friends) but in this revision the scaffolder is using the newly added support for typed APIs via `TypedResults` and `Results<TResult1, TResult2>`.

Reply:
Yes, because `OpenApi` will imply `TypedResults` which needs "Microsoft.AspNetCore.Http.HttpResults"